### DISCUSSION TITLE
Trouble Getting Through First Time Set Up

### DISCUSSION THREAD

**Discussion by Senneseph** [nextcloud/all-in-one#5509] [Questions] [answered]
Hi,
I'm trying to set up the DigitalOcean marketplace solution for Nextcloud on a new Droplet. I must be missing something because when I follow the setup instructions I'm met with the following login screen:

![nextcloud digitalocean login screen](https://github.com/user-attachments/assets/223b9fe7-6583-4234-bdee-e1a8b86f9d5a)

It says "Log in using your Nextcloud AIO passphrase" but I don't know where to get that. I have normal access to the console but would know where to start looking on how to generate a passphrase.

Does anyone have any ideas?

**Comment by szaimen** [comment, marked as answer]
Hi, the AIO passphrase is onky shown the very first time the aio interface is opened. So I wiuld probably recommend to create a new droplet.

**Reply by Senneseph** [reply]
Thank you, I was able to get the passphrase after doing a Rebuild of the same Nextcloud image.

For anyone else running into this issue, this will allow you to preserve IP and DNS settings without having to reset them.  Click into your the Droplet running your Nextcloud instance and choose the "Destroy" menu item.  You should get an option to either Destroy the Droplet or Rebuild Image. I typed "Nextcloud" into the the image selection box and was able to bring up the marketplace image.
